src/main/resources/migrations/master.xml

```diff
@@ -9,5 +9,9 @@
   <include file="migrations/changeset/0003_add_tenant_can_edit_legaltexts_applevel_setting/0003_add_tenant_can_edit_legaltexts_applevel_setting.xml"/>
   <include file="migrations/changeset/0004_add_topic_admin_role_to_super_admin_user/0004_add_topic_admin_role_to_super_admin_user.xml"/>
   <include file="migrations/changeset/0005_add_permission_to_technical_role_rocketchat/0005_add_permission_to_technical_role_rocketchat.xml"/>
+<<<<<<< HEAD
```

Comment:
seems like a merge issue? please test it first on some cluster before merging

Reply:
That is minor and it is removed...